packages/core/upload/admin/src/components/TableList/TableRows.js

```diff
@@ -22,8 +33,16 @@ export const TableRows = ({ onEditAsset, onEditFolder, onSelectOne, rows, select
         const isSelected = !!selected.find((currentRow) => currentRow.id === id);
 
         return (
-          <Tr key={id}>
-            <Td>
+          <Tr
+            key={id}
+            {...onRowClick({
+              fn: () =>
+                elementType === 'asset'
+                  ? onEditAsset(element)
+                  : push(getFolderURL(pathname, query, element)),
+            })}
+          >
+            <Td {...stopPropagation}>
```

Comment:
How does this effect keyboard navigation?

Reply:
<img width="287" alt="image" src="https://user-images.githubusercontent.com/71838159/203503616-d394e8c0-15b3-475e-b461-2c3eb983adc9.png">

this is a utils added during v4 when it was asked to have row click behaviour on `Tables` 
it is used for every Strapi tables right now - without it you cannot click a checkbox or any other button in the table without triggering the `onClick` row (e.g. if you try clicking the checkbox, it would check the checkbox + open edit asset)
I don't know if it would need a refactor at some point but it could be done outside of this feature, wdyt?

Reply:
oh great, it's just onClick propagation. the name eluded to being "all propagation" 👍🏼